http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/RuntimeServiceDbQueries.scala

```diff
@@ -86,7 +86,11 @@ object RuntimeServiceDbQueries {
             // .filter { lbl => (lbl.key, lbl.value) inSetBind labelMap.toSet }
             // Unfortunately slick doesn't support inSet/inSetBind for tuples.
             // https://github.com/slick/slick/issues/517
-            .filter(lbl => labelMap.map { case (k, v) => lbl.key === k && lbl.value === v }.reduce(_ || _))
+            .filter(lbl =>
+              labelMap
+                .map { case (k, v) => lbl.key === k && lbl.value === v }
+                .fold[Rep[Boolean]](false)(_ || _)
```

Comment:
I noticed that this code has a bug in that it would fail if the labelMap passed to it is empty when writing a similar query

Reply:
yea..`reduce` will throw exception when collection is empty